TLM/TLM/UI/WhatsNew/WhatsNew.cs

```diff
@@ -102,7 +102,8 @@ public class WhatsNew {
                     && lineKeyword == MarkupKeyword.VersionStart) {
 
                     var changelog = new Changelog() {
-                        Version = new Version(text),
+                        // version text can be 1.2.3.4 or 1.2.3.4-hotfix-<number>
```

Comment:
Should the end of that comment be `1.2.3-hotfix-<number>` ?

Reply:
probably in the future but now it's `1.2.3.4-hotfix-<number>` -> `11.6.5.2-hotfix-1` and Version accepts only `1.2.3.4` and no text at all